dags/overwatch.py

```diff
@@ -0,0 +1,53 @@
+"""
+Overwatch
+Runs daily at 0700 UTC
+
+Source code is [overwatch repository](https://github.com/mozilla/overwatch-mvp/).
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from operators.gcp_container_operator import GKEPodOperator
+
+docs = """\
+This DAG is executes the Overwatch monitoring system.
+All alerts related to this DAG can be ignored.
+
+(for more info on dim see: https://github.com/mozilla/overwatch-mvp)
+"""
+
+
+default_args = {
+    "owner": "gleonard@mozilla.com",
+    "email": [
+        "gleonard@mozilla.com",
+        "wichan@mozilla.com",
+    ],
+    "start_date": datetime(2022, 11, 1),
```

Comment:
Do you want the DAG to run for each date since the 1st of Nov or it just needs to run for the most recent date?